### PR TITLE
boot up reqKick on windows

### PR DESCRIPTION
**Pull request by niranjan92**
https://github.com/Shippable/node/issues/210

### CODE REVIEW

**Comment by niranjan92** [comment]
```
C:\Users\niranjan>pm2 show 0
 Describing process with id 0 - name shippable-reqKick@6bae8e6b-fd5c-42cd-83ae-150df8b83455.service
┌───────────────────┬─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ status            │ online                                                                                                                      │
│ name              │ shippable-reqKick@6bae8e6b-fd5c-42cd-83ae-150df8b83455.service                                                              │
│ restarts          │ 0                                                                                                                           │
│ uptime            │ 6m                                                                                                                          │
│ script path       │ C:\Users\niranjan\shippable\6bae8e6b-fd5c-42cd-83ae-150df8b83455\reqKick\reqKick.app.js                                     │
│ script args       │ N/A                                                                                                                         │
│ error log path    │ C:\Users\niranjan\shippable\6bae8e6b-fd5c-42cd-83ae-150df8b83455\reqKick\reqKick_6bae8e6b-fd5c-42cd-83ae-150df8b83455.err   │
│ out log path      │ C:\Users\niranjan\shippable\6bae8e6b-fd5c-42cd-83ae-150df8b83455\reqKick\reqKick_6bae8e6b-fd5c-42cd-83ae-150df8b83455.out   │
│ pid path          │ C:\Users\niranjan\shippable\6bae8e6b-fd5c-42cd-83ae-150df8b83455\reqKick\reqKick_6bae8e6b-fd5c-42cd-83ae-150df8b83455-0.pid │
│ interpreter       │ node                                                                                                                        │
│ interpreter args  │ N/A                                                                                                                         │
│ script id         │ 0                                                                                                                           │
│ exec cwd          │ C:\Users\niranjan\shippable\6bae8e6b-fd5c-42cd-83ae-150df8b83455\reqKick                                                    │
│ exec mode         │ fork_mode                                                                                                                   │
│ node.js version   │ 4.8.5                                                                                                                       │
│ watch & reload    │ ✘                                                                                                                           │
│ unstable restarts │ 0                                                                                                                           │
│ created at        │ 2017-11-30T10:02:25.063Z                                                                                                    │
└───────────────────┴─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

File: initScripts/x86_64/WindowsServer_2016/Docker_17.06.ps1 [hunk 1]
```diff
@@ -180,15 +180,46 @@ Function remove_reqProc() {
 }
 
 Function remove_reqKick() {
-  Write-Output "!!! TODO: Remove existing reqKick !!!"
+  Write-Output "Remove existing reqKick"
+
+  pm2 delete all
```
Comment (shrivara) [inline initScripts/x86_64/WindowsServer_2016/Docker_17.06.ps1]: Let's only remove reqKick services. Use regex.

File: initScripts/x86_64/WindowsServer_2016/Docker_17.06.ps1 [hunk 2]
```diff
@@ -180,15 +180,46 @@ Function remove_reqProc() {
...
+  if (Test-Path $REQKICK_CONFIG_DIR) {
```
Comment (shrivara) [inline initScripts/x86_64/WindowsServer_2016/Docker_17.06.ps1]: This is unnecessary as `SHIPPABLE_RUNTIME_DIR` gets removed.

File: initScripts/x86_64/WindowsServer_2016/Docker_17.06.ps1 [hunk 3]
```diff
@@ -109,6 +109,7 @@ Function check_docker_opts() {
 
 Function setup_mounts() {
   if (Test-Path $SHIPPABLE_RUNTIME_DIR) {
+    pm2 stop /shippable-reqKick*/
```
Comment (shrivara) [inline initScripts/x86_64/WindowsServer_2016/Docker_17.06.ps1]: Let's move remove_reqKick up.

File: initScripts/x86_64/WindowsServer_2016/Docker_17.06.ps1 [hunk 4]
```diff
@@ -197,11 +226,11 @@ check_win_containers_enabled
 install_prereqs
 docker_install
 check_docker_opts
+remove_reqKick
 setup_mounts
 setup_envs
 setup_opts
 remove_reqProc
```
Comment (shrivara) [inline initScripts/x86_64/WindowsServer_2016/Docker_17.06.ps1]: Let's move remove_reqProc also up for consistency.

**Pull request merged**